compiler/one-cmds/tests/prepare_test_materials.sh

```diff
@@ -105,6 +105,13 @@ if [[ ! -s "Net_InstanceNorm_003.part" ]]; then
     # https://github.com/Samsung/ONE/issues/8570#issuecomment-1115804257
 fi
 
+if [[ ! -s "Conv2D_000.circle" ]]; then
+    rm -rf Net_InstanceNorm_003.zip
+    wget https://github.com/Samsung/ONE/files/9466423/Conv2D_000.zip
+    unzip Net_InstanceNorm_003.zip
+    # https://github.com/Samsung/ONE/issues/9653#issuecomment-1233668729
+fi
```

Comment:
I uploaded `Conv2D_000.circle` and its output which are generated from `common-artifacts` for test `circle-interpreter` through `one-infer`. Those files are uploaded [here](https://github.com/Samsung/ONE/issues/9653).